src/common/meta/src/version_reporter.rs

```diff
@@ -0,0 +1,200 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::env;
+use std::time::Duration;
+
+use common_runtime::{RepeatedTask, TaskFunction};
+use common_telemetry::info;
+use common_telemetry::tracing::log::warn;
+use once_cell::sync::Lazy;
+use reqwest::{Client, Response};
+use serde::{Deserialize, Serialize};
+
+use crate::error::{Error, Result};
+
+pub const VERSION_REPORT_URL: &str =
```

Comment:
+1. `telemetry` can be a good name for this kind of feature:

https://code.visualstudio.com/docs/getstarted/telemetry
https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/index.html

But it might be confusing with our `common/telemetry` module. Do you have other suggestion?